bank-sdk/example-app/src/main/java/net/gini/android/bank/screen/core/di/GiniExampleModule.kt

```diff
@@ -0,0 +1,82 @@
+package net.gini.android.bank.screen.core.di
+
+import android.content.Context
+import android.text.TextUtils
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import net.gini.android.bank.screen.R
+import net.gini.android.capture.network.GiniCaptureDefaultNetworkService
+import net.gini.android.core.api.DocumentMetadata
+import org.slf4j.Logger
+import javax.inject.Qualifier
+import javax.inject.Singleton
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class GiniCaptureNetworkServiceRelease
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class GiniCaptureNetworkServiceDebug
+
+
+@Module
+@InstallIn(SingletonComponent::class)
+class GiniExampleModule {
+
+    @Singleton
+    @GiniCaptureNetworkServiceRelease
+    @Provides
+    fun bindGiniCaptureNetworkServiceRelease(
+        @ApplicationContext context: Context,
+        logger: Logger
+    ): GiniCaptureDefaultNetworkService {
+        val clientId = context.getString(R.string.gini_api_client_id)
```

Comment:
[Suggestion] These two methods use duplicated code. You could extract them into private methods. But it's not important. You can also leave it as is.

Reply:
Done.